cmd/openshift-install/log.go

```diff
@@ -56,6 +57,20 @@ func (h *fileHook) Fire(entry *logrus.Entry) error {
 		msgs = strings.Split(orig, "\n")
 	}
 
+	level := logrus.InfoLevel
+	if h.config.Level != nil {
+		parsedLevel, err := logrus.ParseLevel(*h.config.Level)
+		if err == nil {
+			level = parsedLevel
+		}
```

Comment:
Do we want to notify on error here? E.g. invalid log level string?
```suggestion
		} else {
			logrus.WithError(err).Debugf("Failed to parse log level %q", *h.config.Level)
		}
```

Reply:
I'll add it. Thanks